core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -561,6 +561,8 @@
           + " to force the changes to be written to disk, the metadata and root tables can be flushed"
           + " and possibly compacted. Legal values are: compact - which both flushes and compacts the"
           + " metadata; flush - which flushes only (compactions may be triggered if required); or none"),
+  GC_ENABLE_METRICS2("gc.enable.metrics2", "true", PropertyType.BOOLEAN,
```

Comment:
Should these be enabled by default when they weren't before?

Reply:
They should be disabled by default - my mistake for not changing back from testing - will restore default before merge.